### ISSUE TITLE
Use semantic versioning

### ISSUE THREAD

**Issue by jdfergason** [damicreabox/Git2Swift#17]
The swift package manager requires that packages use semantic versioning to automatically pull files.  It would be nice if Git2Swift followed this scheme for it's releases:

MAJOR.MINOR.PATCH

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.

The current scheme of 0.X, is missing the PATCH which the package manager does not like

**Comment by damicreabox** [comment]
Yes you right. It's time to create 1.0.0 version. I will use it after I have reviewed your pull request.
